e2e/task/task_suite_test.go

```diff
@@ -27,16 +29,28 @@ var _ = BeforeSuite(func() {
 
 	appName = fmt.Sprintf("e2e-task-%d", time.Now().Unix())
 	envName = "test"
-	// NOTE: cfn stack for task isn't deleted automatically, and `task delete` isn't implemented yet
-	// so we deploy all tasks to the same group, so that it doesn't create a lot of stacks.
 	groupName = fmt.Sprintf("e2e-task-%d", time.Now().Unix())
+	// We name task stack in format of "task-${groupName}".
+	// See https://github.com/aws/copilot-cli/blob/e9e3114561e740c367fb83b5e075750f232ad639/internal/pkg/deploy/cloudformation/stack/name.go#L26.
+	taskStackName = fmt.Sprintf("task-%s", groupName)
+	// We name ECR repo name in format of "copilot-${groupName}".
+	// See https://github.com/aws/copilot-cli/blob/e9e3114561e740c367fb83b5e075750f232ad639/templates/task/cf.yml#L75.
+	repoName = fmt.Sprintf("copilot-%s", groupName)
 })
 
 var _ = AfterSuite(func() {
-	_, err := cli.AppDelete(map[string]string{"test": "default"})
-	Expect(err).NotTo(HaveOccurred())
-
-	// TODO: cli.TaskDelete()
+	// Clean ECR repo before deleting the stack.
+	err := command.Run("aws", []string{"ecr", "delete-repository", "--repository-name", repoName, "--force"})
+	Expect(err).NotTo(HaveOccurred(), "delete ecr repo")
+	// Delete task stack.
+	err = command.Run("aws", []string{"cloudformation", "delete-stack", "--stack-name", taskStackName})
```

Comment:
So I'm trying to understand based on the error that you saw yesterday while trying to delete from console:
```
Role arn:aws:iam::xxx:role/e2e-task-1597338546-test-CFNExecutionRole is invalid or cannot be assumed
```
Will this have the necessary permissions? how come?

Reply:
yeah it will have. I tested that. So it's because we still have the CFN execution role since we haven't deleted the environment yet.

Reply:
Ok thanks that totally makes sense!

I think one thing that these tests made us realize is that if a customer uses the same task group name to run their tasks both in a Copilot env and then in default, they won't be able to delete the stack in case the Copilot environment is deleted which is slightly surprising:

> It is not possible to remove a service role attached to a stack after the stack is created.  

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-iam-servicerole.html

Maybe, when we implement `task delete` we can provide a nicer err message with action items when this err occurs.